inference-engine/samples/common/format_reader/opencv_wraper.cpp

```diff
@@ -40,9 +41,7 @@ std::shared_ptr<unsigned char> OCVReader::getData(size_t width = 0, size_t heigh
 
     size_t size = resized.size().width * resized.size().height * resized.channels();
     _data.reset(new unsigned char[size], std::default_delete<unsigned char[]>());
-    for (size_t id = 0; id < size; ++id) {
-        _data.get()[id] = resized.data[id];
-    }
+    std::memcpy(_data.get(), resized.data, size);
```

Comment:
You could avoid copying entirely by having `resized` use `_data` as its storage.

Reply:
@IRDonch could you please send a patch with your suggestion?

Reply:
I don't have time to write and test a proper patch, but it would look roughly like this:

```c++
if (width == 0) width = img.cols;
if (height == 0) height = img.rows;

size_t size = width * height * resized.channels();
_data.reset(new unsigned char[size], std::default_delete<unsigned char[]>());

cv::Mat resized(height, width, img.type(), _data.get());
cv::resize(img, resized, resized.size());

return _data;